.github/workflows/docker.yml

```diff
@@ -76,7 +76,7 @@ jobs:
             - name: Cross build Lighthouse binary
               run: |
                   cargo install cross
-                  env CROSS_PROFILE=${{ matrix.profile }} make build-${{ matrix.binary }}
+                  env CROSS_PROFILE=${{ matrix.profile }} CROSS_FEATURES=${{ matrix.features.env }} make build-${{ matrix.binary }}
```

Comment:
I think there's another problem waiting here, which is that we'll set the features to `""` in the non-minimal case. I think we need to add the default features from here to preserve the correct behaviour:

https://github.com/sigp/lighthouse/blob/38514c07f222ff7783834c48cf5c0a6ee7f346d0/Makefile#L17-L18

For consistency maybe we should also put the `CROSS_FEATURES` in the `env:` section above (next to features).

Reply:
Amazing catch. I wish all build processes should have been a shell script that is just invoked via Github workflow.

Reply:
I added the default list of features to the docker config, kind of redundant to Makefile configuration, but believe more extensible in future.